app/routes.py

```diff
@@ -1,2 +1,80 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+def validate_id(cls, planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        abort(make_response({"message": f"{cls} {planet_id} invalid"}, 400))
+        
+    query_result = Planet.query.get(planet_id)
+
+    if not query_result:
+        abort(make_response({"message": f"{cls} {planet_id} not found"}, 404))
+
+    return query_result
+
+@planets_bp.route("", methods=["POST"])
+def add_planet():
+    request_body = request.get_json()
```

Comment:
Doing some validation on the request body here (to make sure it has the required fields) would be nice.